automacent-fwk-components/automacent-fwk-core/src/main/java/com/automacent/fwk/core/Driver.java

```diff
@@ -345,9 +352,13 @@ public void startDriver(DriverManagerType driverManagerType) {
 				BaseTest.getTestObject().getTimeoutInSeconds()));
 		webDriver.manage().timeouts().setScriptTimeout(getScriptTimeoutInSeconds(), TimeUnit.SECONDS);
 		_logger.info(String.format("Script timeout set on driver to %s seconds", getScriptTimeoutInSeconds()));
-		webDriver.manage().window().maximize();
-		webDriver.manage().deleteAllCookies();
-		_logger.info("Cookies deleted");
+
+		if (BaseTest.getTestObject().getDebuggerAddress() == null
+				|| !driverManagerType.name().equals(DriverManagerType.CHROME.name())) {

```

Comment:
Change this to 

```
if (BaseTest.getTestObject().getDebuggerAddress() == null) {
}
```

Since we are setting this only if browser is chrome and debugger address is not empty